README.md

```diff
@@ -166,7 +166,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 ### V
 
 * [V8](/glossary/V8.md): Google’s open source javaScript engine. It’s what Chrome is running, but it’s also used for other projects like Node.js and MongoDB.
-* [Vanilla](/glossary/VANILLA.md): *to be completed.*
+* [Vanilla](/glossary/VANILLA.md): Plain JavaScript.
```

Comment:
How about the following?

"A term utilized to describe the use of plain JavaScript without dependencies such as libraries and/or frameworks."